packages/ymax-planner/src/axelar/gmp-status.ts

```diff
@@ -0,0 +1,346 @@
+import { ethers } from 'ethers';
+const wait = async (seconds: number) => {
+  return new Promise(resolve => setTimeout(resolve, seconds * 1000));
+};
+
+type TxReceipt = {
```

Comment:
Consider moving these types into a separate .d.ts file for focusing the purpose of this file.

Reply:
since we can use `.ts` let's not manually create `.d.ts` files. the build tools will do that as necessary